manim/scene/scene.py

```diff
@@ -1264,19 +1265,19 @@ def check_interactive_embed_is_valid(self):
                 "Disabling interactive embed as 'skip_animation_preview' is enabled",
             )
             return False
-        elif config["write_to_movie"]:
```

Comment:
unrelated changes ?

Reply:
Yes, I figured that I may as well add small touch ups and fixes as I go through all the other changes I made. I'm happy to separate this into another PR if you'd prefer. 

These changes are just to bring the code base in line with [PEP8](https://peps.python.org/pep-0008/). 

From the aforementioned link, consider:

```python
For sequences, (strings, lists, tuples), use the fact that empty sequences are false:

# Correct:
if not seq:
if seq:

# Wrong:
if len(seq):
if not len(seq):
```

For a mathematical perspective, we can think of non-empty sequences as an equivalence class. Similarly, the non-zero values are an equivalence class.